tests/std/tests/P1502R1_standard_library_header_units/test.cpp

```diff
@@ -68,6 +68,7 @@ import <span>;
 import <spanstream>;
 import <sstream>;
 import <stack>;
+import <stacktrace>;

```

Comment:
Ideally, we'd additionally add a block below to exercise `<stacktrace>` machinery at runtime, below the `<stack>` test:

https://github.com/microsoft/STL/blob/4bb41db3d8e7605fc03656659752f3fad4133bc0/tests/std/tests/P1502R1_standard_library_header_units/test.cpp#L745-L759

However, due to `<stacktrace>`'s special requirements (needing a PDB etc.), it may be difficult to do so in this test. @michaelbprice suggests that a default-constructed `stacktrace_entry` might not have PDB requirements - that would be perfectly fine to test (as here we're just trying to exercise a little bit of each header). If that's not feasible, it would be reasonable to skip automated coverage of `<stacktrace>` here, but in that case, we should add a comment below along the lines of `// not testing <stacktrace> for now, due to its PDB requirements`.

Reply:
It is feasible to construct even normal, not default-constructed `stacktrace_entry`.
Moreover, if we use `__declspec(dllexport)` functions, they will be seen in the trace, even if .pdb is missing, so we can check function names.
Sure file/line can be tested only with .pdb .
How far the test should go?

Reply:
I don't think the test has to go too far. Even something as simple as showing that when imported as a module, you can even reference any of the exported names would be better than nothing. It looks like most of the existing tests go further than that and test behavior, but those other APIs are a lot easier create contrived examples for that have no external dependencies.